### PR TITLE
[WIP] feature/cache-all-s3-calls-in-after-request

### PR DESCRIPTION
**Pull request by pkfec**

### CODE REVIEW

**Comment by pkfec** [comment]
@LindsayYoung  : Please review the PR.


cc : @hcaofec

**Comment by pkfec** [comment]
In my local build a new virtual env on `openFEC` repo, ran the `py.test` and submitted the PR. 
@LindsayYoung : Please review the latest PR.

cc @hcaofec

**Comment by pkfec** [comment]
- Added a new env variable "CACHE_ALL_REQUESTS" in manifest-base.yml.  This env variable is set to `True` which triggers the after_requests, which caches (saves in s3) all the web requests.
- Using "regex" , removed the api_key from the URL.
- Updated the test_utils.py :  remove the api_key and special characters from the URL

@LindsayYoung : Please review
cc @hcaofec

File: manifests/manifest_api_dev.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 inherit: manifest_dev.yml
 env:
   NEW_RELIC_APP_NAME: fec | api | dev
+  CACHE_ALL_REQUESTS: false
```
Comment (LindsayYoung) [inline manifests/manifest_api_dev.yml]: Let's get rid of this line for now so we can test it on dev

File: tests/test_utils.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import unittest
+import re
```
Comment (LindsayYoung) [inline tests/test_utils.py]: remove

File: webservices/rest.py [hunk 3]
```diff
@@ -17,6 +17,9 @@ def initialize_newrelic():
 import os
 import http
 import logging
+import json
+import boto
+import re
```
Comment (LindsayYoung) [inline webservices/rest.py]: moved this one too

**Comment by LindsayYoung** [comment]
Thank you so much! This is such a great start

**Pull request merged**